octobot/src/assets/app.js

```diff
@@ -353,6 +353,7 @@ app.controller('ReposController', function($rootScope, $scope, sessionHttp, noti
   $scope.addRepo = function() {
     $scope.theRepo = {
       force_push_notify: true,
+      use_threads: false,
```

Comment:
should update the default here too

```suggestion
      use_threads: true,
```